sql/catalyst/src/main/scala/org/apache/spark/sql/types/CalendarIntervalType.scala

```diff
@@ -22,11 +22,14 @@ import org.apache.spark.unsafe.types.CalendarInterval
 
 /**
  * The data type representing calendar time intervals. The calendar time interval is stored
```

Comment:
calendar **time** intervals . It is arguable but it can represent date interval too.